Server/controllers/time-entry.controller.js

```diff
@@ -4,13 +4,124 @@
  *
  * @class timeEntriesController
  */
+const mongoose = require('mongoose');
+const timeEntryModel = require('../models/time-entry.model');
+const dataLayer = require('../datalayer/time-entry.datalayer')
+const moment = require('moment');
+
+
 class timeEntriesController {
 
-    static getTimeEntriesByUserId (req, res) {}
+    //  GET 
+
+    static async getTimeEntriesByUserId(req, res) {
+        let timeEntries;
+
+        const obj = {
```

Comment:
We can use Obejct destructuring for this also https://hacks.mozilla.org/2015/05/es6-in-depth-destructuring/

Reply:
Updated

Reply:
This is valid but not destructuring. Please change to in the form of e.g.
`const {user, limit, sort} = req.query`

Reply:
Updated